src/tusb_option.h

```diff
@@ -27,6 +27,9 @@
 #ifndef _TUSB_OPTION_H_
 #define _TUSB_OPTION_H_
 
+// To avoid GCC compiler warnings when -pedantic option is used (strict ISO C)
+typedef int make_iso_compilers_happy ;
```

Comment:
why is this necessary? what was the error?

Reply:
This is not an error correction.
Only good programming practice: avoid displaying dozens of warning messages when compiling these modules.
If a warning appears, you have to go see what it is, and maybe an important warning is buried in it.

Reply:
sorry, I meant what warning this was correcting